src/main/java/com/joss/conductor/mobile/Locomotive.java

```diff
@@ -330,38 +331,10 @@ public boolean isPresentWait(String id) {
     }
 
     public boolean isPresentWait(By by) {
-
-        //Line Separator Variable for formatting output
-        String newLine = System.getProperty("line.separator");//This will retrieve line separator dependent on OS.
-        //Array of stacktrace elements to output
-        StackTraceElement[] stackTraceElements = Thread.currentThread().getStackTrace();
-
         try {
-            waitForCondition(ExpectedConditions.not(ExpectedConditions.invisibilityOfElementLocated(by)));
-
+            isPresent(by);
```

Comment:
Don't think this additional check here is actually necessary, it calls `getAppiumDriver().findElements(by).size();` below which is the same impl as `isPresent()`.

Reply:
What are your thoughts on replacing the whole method with 

`waitForCondition(visibilityOfAllElementsLocatedBy(by));`

I'm not sure of the purpose or intent of `isPresentWait`, what are the benefits of its current implementation?

Reply:
I'd suggest that approach, we should move away from `Thread.sleep()`s if possible, it seems like the conditional waits provided by appium should be pretty good to use.

IIRC this method was just implemented to have some sort of wait condition for the `isPresent` method, since `isPresent` runs immediately there are race condition implications to using it.

Reply:
Thoughts on 2845ba2 ?

Reply:
Added option to configure timeout with https://github.com/willowtreeapps/conductor-mobile/pull/113/commits/11243c8c61f8282ab5f4866eb05cc78b0c2e9b69